runtime/gc_base/GCExtensions.cpp

```diff
@@ -267,3 +273,65 @@ MM_GCExtensions::computeDefaultMaxHeap(MM_EnvironmentBase *env)
 
 	memoryMax = MM_Math::roundToFloor(heapAlignment, memoryMax);
 }
+
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+#if !defined(LINUX)
```

Comment:
I would add to this check also:
 - fail if this is not a 64-bit JVM `defined(J9VM_ENV_DATA64)`
 - fail if this is ZTPF `defined(OMRZTPF)`